### PR TITLE
getStudentSections implementation

### PR DESCRIPTION
**Pull request by jrm86**
Mirrors functionality of the getInstructorStudents() functions, but for sections which students have attended.

### CODE REVIEW

**Comment by jrm86** [comment]
Just altered GRANT EXECUTE to allow students to perform the function (to view their own sections).

**Review by DasilvaB — APPROVED** [review]
Looks good.

File: src/db/addStudentMgmt.sql [hunk 1]
```diff
@@ -462,4 +462,99 @@ GRANT EXECUTE ON FUNCTION assignFinalGrade(student INT, sectionID INT,
...
+                                          )
+   FROM PUBLIC;
+
+GRANT EXECUTE ON FUNCTION getStudentSections(studentID INT,
+                                          year NUMERIC(4,0),
+                                          seasonOrder NUMERIC(1,0),
+                                          courseNumber VARCHAR(8)
+                                          )
+   TO alpha_GB_Webapp, alpha_GB_Instructor, alpha_GB_Registrar, 
+      alpha_GB_RegistrarAdmin, alpha_GB_Admissions, alpha_GB_DBAdmin,
```
Comment (afig) [inline src/db/addStudentMgmt.sql]:
```suggestion
   alpha_GB_RegistrarAdmin, alpha_GB_Admissions, alpha_GB_DBAdmin,
```

Small indentation fix

File: src/db/addStudentMgmt.sql [hunk 2]
```diff
@@ -462,4 +462,99 @@ GRANT EXECUTE ON FUNCTION assignFinalGrade(student INT, sectionID INT,
...
+      alpha_GB_Student;
```
Comment (afig) [inline src/db/addStudentMgmt.sql]:
```suggestion
   alpha_GB_Student;
```

File: src/db/addStudentMgmt.sql [hunk 3]
```diff
@@ -462,4 +462,99 @@ GRANT EXECUTE ON FUNCTION assignFinalGrade(student INT, sectionID INT,
...
+
+REVOKE ALL ON FUNCTION getStudentSections(studentID INT,
+                                          year NUMERIC(4,0),
+                                          seasonOrder NUMERIC(1,0),
+                                          courseNumber VARCHAR(8)
+                                          )
+   FROM PUBLIC;
+
+GRANT EXECUTE ON FUNCTION getStudentSections(studentID INT,
+                                          year NUMERIC(4,0),
```
Comment (afig) [inline src/db/addStudentMgmt.sql]:
```suggestion
                                             year NUMERIC(4,0),
```

In general, all parameters should line up with the first parameter (like on lines 543-546). I would repeat this with the other parameter lists added.

**Review by afig — APPROVED** [review]
Overall the new functions look great @jrm86. Thanks!

I added a few comments about indentation inconsistencies. If you have time it would be nice to address them, but not essential.

File: src/db/addStudentMgmt.sql (hunk 3 above)
Comment (jrm86) [reply src/db/addStudentMgmt.sql]: Good catch, thanks!

**Pull request merged**